apps/docs/stories/components/PostalCodeForm/Docs.mdx

```diff
@@ -0,0 +1,47 @@
+import { Meta, Title, ArgTypes, Source } from '@storybook/blocks';
+import { filterDocsByTag, ExportedParts } from '../../utils';
+import { codeExampleFull } from './example';
+import { ComponentBox } from '../../utils/documentation-components';
+import * as JustifiPostalCodeForm from './index.stories';
+import { setUpMocks } from '../../utils/mockAllServices';
+
+{setUpMocks()}
+
+<Meta
+  title="Modular Checkout/Postal Code Form"
+  Components="justifi-postal-code-form"
+  of={JustifiPostalCodeForm}
+/>
+
+<Title />
+
+---
+
+Renders a form for collecting the postal code as part of a checkout flow.
+
+This component is **designed to be used within** the `justifi-modular-checkout` and **does not accept props directly**. Instead, it relies on shared state passed through the Stencil Store, managed by the wrapper component.
```

Comment:
Change to '... relies on the shared state...' and ' managed by the `justifi-modular-checkout` component.'